utils/colors.js

```diff
@@ -0,0 +1,7 @@
+const colors = {
```

Comment:
Maybe it would be better to store this in `/styles/colors.js` instead of `/utils/colors.js`